targets/TARGET_NUVOTON/TARGET_M480/trng_api.c

```diff
@@ -77,21 +82,21 @@ void trng_free(trng_t *obj)
 int trng_get_bytes(trng_t *obj, uint8_t *output, size_t length, size_t *output_length)
 {
     (void)obj;
-
-    *output_length = 0;
-    if (length < 32) {
-        unsigned char tmpBuff[32];
+    unsigned char tmpBuff[32];
```

Comment:
Consider making a define for `32` to use for the size of `tmpBuff`.

Reply:
@Patater  Update the code in new commit d8a9e35